### ISSUE TITLE
User Experience in Olivera

### ISSUE THREAD

**Issue by kstapelfeldt** [Islandora/documentation#1990]
Change default colour scheme to white
Add new logo image

**Comment by kstapelfeldt** [comment]
![Screen Shot 2021-11-30 at 2 47 39 PM](https://user-images.githubusercontent.com/467898/144117401-1ec0d860-42d9-4ded-b6b4-7a6549eacffe.png)

**Comment by DonRichards** [comment]
Is Olivera being added to the default setup?

**Comment by kstapelfeldt** [comment]
no, sorry. Probably this isn't the right place for this ticket. Let's talk in tomorrow's meeting about what we're proposing as part of the default experience group.

**Comment by alxp** [comment]
Per the kick-off meeting it was decided we would start out with the Islandora Starter Theme along with the rest of the install profile and then once that is in place see how it looks when we switch themes.

**Comment by kstapelfeldt** [comment]
Sorry if this causes confusion. It's just notes to myself and Nat and I'm dumping them in this ticket.
![Screen Shot 2021-11-30 at 2 58 14 PM](https://user-images.githubusercontent.com/467898/144118906-8ec86cfa-2cf5-4278-85cf-a79dda4ca58b.png)
Search bar has title baked in a weird way and I can't edit "Fulltext search" or get rid of it like I expect.

**Comment by kstapelfeldt** [comment]
Name changed to 'Sandbox'

**Comment by kstapelfeldt** [comment]
turned off title for search bar and assigned to sidebar. 
Created new default homepage and assigned in configuration.
Excluded page title from the front page (<front)
